EleksTubeHAX_pio/src/main.cpp

```diff
@@ -295,16 +304,22 @@ void loop() {
         time_t offset = uclock.getTimeZoneOffset();
         int8_t offset_hour = offset/3600;
         int8_t offset_min = (offset%3600)/60;
+        if(offset_min < 0) {
+          offset_min = -offset_min;
+        }
```

Comment:
Do you want the negative?  Or `+60` here?  When you're talking about modulo, I think `+60` is correct.

Reply:
What is needed here is just removing the negative sign in front of the minutes.
If you want to show time offset of "-2 and 3/4" hours is this -2h and 45 min. Calculation will show you "-2:-45" and we want "-2:45". 
On the other hand -45 + 60 = 15 which is not a correct indication.